src/Avalonia.FuncUI.ControlCatalog/Views/Tabs/MaskedTextBoxDemo.fs

```diff
@@ -0,0 +1,72 @@
+namespace Avalonia.FuncUI.ControlCatalog.Views
+
+open System
+open Avalonia.Controls
+open Elmish
+open Avalonia.FuncUI
+open Avalonia.FuncUI.DSL
+open Avalonia.FuncUI.Components
+open Avalonia.FuncUI.Elmish
+open Avalonia.Input
+open Avalonia.Interactivity
+open Avalonia.Layout
+
+module MaskedTextBoxDemo =
+    type State = { count: int }
+
+    let init = { count = 0 }
+
+    type Msg =
+    | SetCount of int
+
+    let update (msg: Msg) (state: State) : State =
+        match msg with
+        | SetCount number -> { state with count = number }
+           
+    let view (state: State) (dispatch) =
+        StackPanel.create [
+            StackPanel.spacing 10.0
+            StackPanel.children [
+                TextBox.create [
+                    TextBox.text (string state.count)
+                    TextBox.onTextChanged (fun text ->
+                        let isNumber, number = Int32.TryParse text
+                        if isNumber then
+                            dispatch (SetCount number)
+                            
+                    , SubPatchOptions.Never) // <- this is annoying.
```

Comment:
Yeah, I think we should open a separate issue to track things like these that need to be documented and go for some defaults